README.md

```diff
@@ -1,6 +1,5 @@
 [![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](LICENSE.txt)
 [![Build Status](https://dev.azure.com/scipp/scipp/_apis/build/status/master?branchName=master)](https://dev.azure.com/scipp/scipp/_build?definitionId=1)
-[![Documentation Status](https://readthedocs.org/projects/scipp/badge/?version=latest)](https://scipp.readthedocs.io/en/latest/?badge=latest)
```

Comment:
Maybe we can have a badge for the pages docs?
Also, should probably update the `readthedocs` link just below in the Scipp description.

Reply:
A badge is not needed, it is covered by the pipeline badge.
Links will be updated once the deployment is confirmed to be working.